backend/tests/Feature/Material/NetRequirementsTest.php

```
@@ -0,0 +1,230 @@
+
+        $report = $this->service->report($this->from, $this->to, $line->id);
+        $row = $this->rowFor($report, $material->id);
+
+        $this->assertEqualsWithDelta(100, $row['required_qty'], 0.001); // only the in-window, on-line WO
+        $this->assertSame(1, $report['totals']['work_orders']);
+    }
+
+    public function test_api_endpoint_returns_report_for_admin(): void
+    {
+        $this->seed(\Database\Seeders\RolesAndPermissionsSeeder::class);
+        $admin = User::factory()->create();
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\User::factory().


(staticMethod.notFound)